parm/config/config.fv3

```diff
@@ -101,6 +105,17 @@ case $case_in in
         export WRITE_GROUP_GFS=${WRITE_GROUP_GFS:-2}
         export WRTTASK_PER_GROUP_GFS=${WRTTASK_PER_GROUP_GFS:-$npe_node_max}
         export WRTIOBUF=${WRTIOBUF:-"16M"}
+        if [[ $machine == "ORION" ]]; then
+            export OUTPUT_FILETYPE_SFC="netcdf_parallel"
+        fi
+        if [[ $machine == "WCOSS_DELL_P3" ]]; then
+            RESTILE=$(echo $CASE |cut -c 2-)
+            export ichunk2d=$((4*RESTILE))
```

Comment:
May I ask what is the reason to choose these specific chunksizes? What benefit do we get?

Reply:
They are just being moved from `config.fcst` to `config.fv3`

Reply:
All chunk settings were pulled either from what was already being used in the EnKF or free forecast previously.